extensions/interactions/CodeRepl/protractor.js

```diff
@@ -41,9 +41,11 @@ var expectInteractionDetailsToMatch = function(elem, placeHolderText) {
 };
 
 var submitAnswer = function(conversationInput, answerCode) {
+  /* eslint-disable quotes */
```

Comment:
@rohitkatlaa No need to disable quotes if there is no linting problem. I think the linter automatically recognizes this situation where you need to bring in double quotes because the strings have single quotes. Please revert this change, thanks!

Reply:
Done.